src/main/java/appeng/client/render/crafting/MolecularAssemblerRenderer.java

```diff
@@ -139,22 +143,30 @@ private void renderStatus(MolecularAssemblerTileEntity molecularAssembler, Matri
     }
 
     /**
-     * See above for when this can be removed. It creates a RenderType that is equivalent to
-     * {@link RenderType#getTranslucent()}, but enables alpha testing. This prevents the fully transparents parts of the
-     * rendered block model from occluding our particles.
+     * This class gives us access to the protected RenderState.TRANSLUCENT_TRANSPARENCY field.
```

Comment:
It is somewhat obvious that extending another class allows it to access protected fields or methods.

Reply:
Moving the documentation about the workaround here is probably more fitting than hiding it in some method. When coming back to it later and seeing there is a custom `RenderType` and asking yourself why it even exists, it is better to have the documentation here.

Reply:
Is it that hard to read the documentation of the only method in this class (besides the constructor)? We have bigger fishes to fry imo.

Reply:
Added the explanation back in, but the comment about why we use a subclass is fine. BTW forge does the same thing (ForgeRenderTypes class)